sys/include/net/sixlowpan/nd.h

```diff
@@ -260,6 +260,19 @@ static inline void sixlowpan_nd_opt_abr_set_version(sixlowpan_nd_opt_abr_t *abr_
     abr_opt->vhigh = byteorder_htons((uint16_t)(version >> 16));
 }
 
+/**
+ * @brief   Get the lifetime of an Authoritative Border Router from an ABR option
+ *
+ * @param[in] abr_opt   An Authoritative Border Router option (ABRO).
+ *
+ * @return  The lifetime of this ABR in minutes
```

Comment:
```suggestion
 * @return  The lifetime of @p abr_opt in minutes. If the lifetime is 0,
 *          @ref SIXLOWPAN_ND_OPT_ABR_LTIME_DEFAULT is returned.
```

Alternative "ABR option" instead of "ABR", as this is referring to the option, not the border router.

Reply:
done